packages/FileUpload/src/index.tsx

```diff
@@ -168,18 +168,20 @@ const FileUpload: React.SFC<IFileUploadProps> = ({
 
   const handleInputChange = ({ event }: any) => {
     const {
-      target: { id, name, value }
+      target: { id: _id, name, value }
     } = event;
 
     const filesToUploadCopy = cloneDeep(filesToUpload);
+
     const previewImagesCopy = cloneDeep(previewImages);
+    const position = _id.split("-")[0];
```

Comment:
here I would use finIndex high order function and I would find by id the position if u change the previous id and remove the index.